<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentLibrary</class>
 <widget class="QWidget" name="studentLibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #ebf5df; /* Fond de la fenêtre */
}</string>
  </property>
  <widget class="QTableView" name="bookListView">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>501</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background: white;
    color: rgb(85, 0, 127);
    font-size: 10px;
    font: 600 11pt &quot;Segoe UI&quot;;
}QTableView {
    color: rgb(104, 0, 156);
    font: 700 9pt &quot;Segoe UI&quot;;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookDetailsText">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
    background: white;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#55007f;&quot;&gt;Label de détails ::&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>171</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: #7A28A5; /* ou une autre couleur */
border-radius: 8px;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="backButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>240</y>
         <width>169</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string>Retourn</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnVoirCarte">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>169</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string>Map</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addToCartButton">
       <property name="geometry">
        <rect>
         <x>-40</x>
         <y>10</y>
         <width>258</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string>Add to cart</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnHistorique">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>169</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string>Historique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCart">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>50</y>
         <width>201</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string>Cart</string>
       </property>
      </widget>
      <widget class="QPushButton" name="borrowButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>169</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string>Borrow</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>0</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background: #7A28A5;
    border-radius: 20px;
    padding: 10px;
    color: white;
    font-size: 14pt;
}
QLineEdit:focus {
    border: 2px solid #ffd700; /* Ajouter une bordure dorée lorsque le champ de texte est actif */
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
 
    background: #ffd700;
    color: white;
    font-size: 20px;
    font: 63 14pt &quot;Georgia&quot;;
    padding: 10px; border-radius: 10px;
}

QPushButton:hover {
    background: #8b00cc; /* Couleur au survol des boutons */
}

QPushButton:pressed {
    background: #7500a0; /* Couleur lorsque le bouton est cliqué */
}



</string>
      </property>
      <property name="text">
       <string>Searche</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
